commands/project/add.ts

```diff
@@ -142,5 +147,18 @@ exports.builder = yargs => {
     ],
   ]);
 
-  return yargs;
+  return yargs as Argv<ProjectAddArgs>;
+}
+
+export const builder = makeYargsBuilder<ProjectAddArgs>(
+  projectAddBuilder,
+  command,
+  describe!
+);
+
+module.exports = {
```

Comment:
Do we need this? Aren't the individual exports enough?

Reply:
Once I port `commands/projects.ts` to TS I'll be able to remove all of these `module.exports`. The issue is that the project.ts file is still commonJS and using requires, which means I'll need to keep these until I port that file. But I'm going to do that as soon as I finish up the rest of the project commands.